src/index.js

```diff
@@ -132,18 +121,35 @@ class TopLevelContainer extends React.Component {
   constructor(props) {
     super(props);
 
+    this.moveCardToMainboard = this.moveCardToMainboard.bind(this);
+    this.moveCardToSideboard = this.moveCardToSideboard.bind(this);
     this.state = {
       boardState: new BoardState(INITIAL_CARD_NAMES, NUM_COLS),
       sideboardState: new BoardState([], 1),
     };
   }
 
+  moveCardToSideboard(card) {
```

Comment:
I think I'd prefer to define these functions inside the `render` function, since they don't need to be used anywhere else. You can use syntax like:
```
const moveCardToMainboard = (card) => {
...
}
```

I think that'll also remove the need for the extra lines in the constructor.

Reply:
Hmm, that makes sense. And I think you're right that it will remove the need for binding as a result of using the => syntax.